.githooks/pre-commit/test

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env node
 
-var spawn = require('child_process').spawn;
-var exec = require('child_process').execSync;
-var gitCachedFiles = exec('git diff --cached --name-only --diff-filter=ACMR').toString().trim();
+var CP = require('child_process');
+var gitCachedFiles = CP.execSync('git diff --cached --name-only --diff-filter=ACMR').toString().trim();
+var opts = {stdio: 'inherit'};
 
 if (gitCachedFiles) {
-    spawn('npm', ['test'], {stdio: 'inherit'}).on('close', process.exit.bind(process));
+    (process.platform === 'win32' ?
+        CP.exec('npm test', opts) : CP.spawn('npm', ['test'], opts)).on('close', process.exit.bind(process));
```

Comment:
exec().stdout.on('data') resolves problem. There is error occur if i spawn npm(npm enoent).